patcher/src/widgets/Controller/components/CharacterSelect/components/CharacterSelectList.tsx

```diff
@@ -131,7 +131,10 @@ class CharacterSelectList extends React.Component<CharacterSelectListProps, Char
       return canAccessChannel(patcher.getPermissions(), server.channelPatchPermissions);
     });
     const sortedServers = selectedServer ?
-      _.sortBy(serversForPermission, server => server.shardID === selectedServer.shardID ? -1 : 0) :
+      _.sortBy(serversForPermission, [
```

Comment:
The plan for the sorting of these servers is actually ordered by the last characters logged in. I think we save these logged in values somewhere in localStorage.

Reply:
Yep, under PatchButton/index.tsx when we call PlayNow.
```
if (selectedServer) lastPlay.serverName = selectedServer.name;
if (selectedServer.selectedCharacter) lastPlay.characterID = selectedServer.selectedCharacter.id;
localStorage.setItem('cse-patcher-lastplay', JSON.stringify(lastPlay));
```